### PR TITLE
Update buildTools, androidCompileSdk, androidBuildTools and androidSupportLibrary to the latest version:

### PR DESCRIPTION
**Pull request by humazed**
buildToolsVersion to <25.0.2>
 androidCompileSdkVersion to <25>
 androidBuildToolsVersion to <25.0.2>
 androidSupportLibraryVersion to <25.2.0>

as the previous one were very old

### CODE REVIEW

**Comment by philipwhiuk** [comment]
Some of these are possibly fine. But we can't just update targetSdk without implementing stuff affected by changes to Android in the intervening versions.

See #2108 for more.

**Comment by humazed** [comment]
Ok... then should I a new pull request without changing the targetSdk version.

**Comment by philipwhiuk** [comment]
Once you've looked at the effect and changes in the other ones, sure.

**Comment by ArchangeGabriel** [comment]
This looks to me as bumping for the sake of bumping… I’m not a dev here (neither an Android app one btw), but generally when such changes are proposed I like to know what they bring with them, i.e. why should the version be updated and what eventual cons they are.

The situation with targetSdk for instance leads one to think there might be a lot of things lurking there for the other ones too.

**Comment by humazed** [comment]
When someone imports the project into the last version of the android studio it will give him errors about using old androidBuildToolsVersion and will ask you to update them,
and the new androidBuildTools bring improvements in build speed.
 and I see no harm in updating androidCompileSdk, androidBuildTools, and androidSupportLibrary.

**Comment by philipwhiuk** [comment]
We should update them. After reviewing the changes.

Please remove the references to build tools you added to individual projects.

That's the point of the gradle.properties file. It reduces the amount of dependency management we have to do.

Also please confirm you have tested the app and looked at the changes in the ones you are updating.

**Comment by humazed** [comment]
Ok... I have tested the app and it works the same as before my changes.

and I will remove the references to build tools, I left the because I find them this way and thought you left them for a reason.

File: plugins/openpgp-api-lib/openpgp-api/build.gradle [hunk 1]
```diff
@@ -16,6 +16,7 @@ android {
     lintOptions {
         abortOnError false
     }
+    buildToolsVersion androidBuildToolsVersion
```
Comment (philipwhiuk) [inline plugins/openpgp-api-lib/openpgp-api/build.gradle]: Duplicate of line 6 which was already present, remove.

File: plugins/HoloColorPicker/build.gradle [hunk 2]
```diff
@@ -16,4 +16,5 @@ android {
     lintOptions {
         abortOnError false
     }
+    buildToolsVersion androidBuildToolsVersion
```
Comment (philipwhiuk) [inline plugins/HoloColorPicker/build.gradle]: Duplicate of line 5 which was already present, remove.

File: k9mail/build.gradle [hunk 3]
```diff
@@ -103,6 +103,7 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_7
         targetCompatibility JavaVersion.VERSION_1_7
     }
+    buildToolsVersion androidBuildToolsVersion
```
Comment (philipwhiuk) [inline k9mail/build.gradle]: Duplicate of line 49 which was already present, remove.

File: k9mail-library/build.gradle [hunk 4]
```diff
@@ -67,4 +67,5 @@ android {
         exclude 'META-INF/NOTICE.txt'
         exclude 'LICENSE.txt'
     }
+    buildToolsVersion androidBuildToolsVersion
```
Comment (philipwhiuk) [inline k9mail-library/build.gradle]: Duplicate of line 35 which was already present, remove.

**Comment by cketti** [comment]
I tried to update the branch. But that didn't work. New PR here: #2386

**Pull request closed without merging**